tiledb/sm/misc/tdb_time.cc

```diff
@@ -44,9 +44,9 @@ namespace tiledb::sm::utils::time {
 
 uint64_t timestamp_now_ms() {
 #ifdef _WIN32
-  struct _timeb tb;
+  struct __timeb64 tb;
   memset(&tb, 0, sizeof(struct _timeb));
```

Comment:
If you switch from `struct _timeb` to `struct __timeb64` the `memset()` may need adjusting too.